AmplifyPlugins/Core/AWSPluginsCoreTests/Model/Decorator/AuthRuleDecorator/ModelMultipleOwnerAuthRuleTests.swift

```diff
@@ -0,0 +1,298 @@
+//
+// Copyright 2018-2020 Amazon.com,
+// Inc. or its affiliates. All Rights Reserved.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+import XCTest
+@testable import Amplify
+@testable import AmplifyTestCommon
+@testable import AWSPluginsCore
+/*
+ type ModelMultipleOwner @model
+   @auth(rules: [
+
+     # Defaults to use the "owner" field.
+     { allow: owner },
+
+     # Authorize the update mutation and both queries. Use `queries: null` to disable auth for queries.
```

Comment:
both queries? do you mean read queries?

Reply:
good question, i copied that example from https://docs.amplify.aws/cli/graphql-transformer/directives#owner-authorization .. going to remove it from our comment for now.. until i re-read the documentation about that